### PR TITLE
Added Inputs

### PR DESCRIPTION
**Pull request by atharvabhanage02**
Added Text , Number , Password , Success , Warning , Error , Radio and Checkbox Input Fields
Output is in the README file

### CODE REVIEW

File: components/components.css [hunk 1]
```diff
@@ -55,4 +55,10 @@
      --badge-count-color : #eeebeb;
      --badge-count-border-radius: 50%;
      /* Badges End */
+
+     /* Inputs */
+    --input-font-size:1rem;
+    --text-grey:#808080;
+    --input-width:300px;
```
Comment (JantuDeb) [inline components/components.css]: Try to avoid using fixed width for input, you may need it in different places where you require different width.

**Review by JantuDeb — APPROVED** [review]
Suggested some changes.

**Review by sks-21 — APPROVED** [review]
The code looks nice. I would suggest you to prefer relative units over absolute units.

File: components/inputs/inputs.css [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+.apex-input-error{
+    border: 2px solid var(--dark-red);
+}
+
+/* Radio and Checkbox Input */
+.input-select{
+    font-size: 1.25rem;
+}
+.radio-input,.checkbox-input{
+    height: 15px;
```
Comment (t007rushi) [inline components/inputs/inputs.css]: Use relative units like rem

File: components/components.css (hunk 1 above)
Comment (atharvabhanage02) [reply components/components.css]: Yes , got it . I'll make the changes
Thanks for correcting me

File: components/inputs/inputs.css (hunk 2 above)
Comment (atharvabhanage02) [reply components/inputs/inputs.css]: Yes, I'll do that . Thanks for suggesting that .

**Pull request merged**